Rakefile

```diff
@@ -2,8 +2,16 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path("config/application", __dir__)
-if Rails.env.development? || Rails.env.test?
+
+Rails.application.load_tasks
+
+begin
   require "ci/reporter/rake/minitest"
+  require "rubocop/rake_task"
+  RuboCop::RakeTask.new
+rescue LoadError
+  # Tasks aren't available in all environments
 end
 
-Rails.application.load_tasks
+Rake::Task[:default].clear if Rake::Task.task_defined?(:default)
```

Comment:
Curious how come we need this?

Reply:
Sure, I've just added details to the commit message (I got a bit lazy applying it all the other place) 

Rails' test task and cucumber set themselves as the default task (if they're loaded - which doesn't happen in prod, hence the conditional) and rather than wrangling that it seemed more explicit to clear and start again.

Your comment made me realise I'd missed `cucumber` from the task 🙈